<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <layout class="QVBoxLayout" name="LeftLay">
        <item>
         <widget class="QToolBox" name="LeftTool">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="InfoPage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>439</width>
             <height>431</height>
            </rect>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <attribute name="label">
            <string>绘图信息</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QDoubleSpinBox" name="XmaxSpinBox">
                <property name="maximum">
                 <double>10000.000000000000000</double>
                </property>
                <property name="value">
                 <double>1023.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>区域最大值</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QDoubleSpinBox" name="XminSpinBox">
                <property name="maximum">
                 <double>10000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>区域最小值</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>整体最大计数：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="MaxCount">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>整体最小计数：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="MinCount">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QCheckBox" name="TPFBox">
                <property name="text">
                 <string>三点平滑</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QCheckBox" name="FPFBox">
                <property name="text">
                 <string>五点平滑</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QCheckBox" name="HAFBox">
                <property name="text">
                 <string>高精度模式</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="LogXBox">
                <property name="text">
                 <string>对数横坐标</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QCheckBox" name="LogYBox">
                <property name="text">
                 <string>对数纵坐标</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QCheckBox" name="DotBox">
                <property name="text">
                 <string>点状显示</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QCheckBox" name="AutoBox">
                <property name="text">
                 <string>自动量程</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="ProcessPage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>439</width>
             <height>431</height>
            </rect>
           </property>
           <attribute name="label">
            <string>数据处理</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="MidLay">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="RightLay" stretch="0,0,0">
         <item>
          <widget class="QTimeEdit" name="timeEdit">
           <property name="displayFormat">
            <string>HH:mm:ss.zzz</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="ThreadBox">
           <property name="text">
            <string>开启时间线程</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>绘图</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCtrl">
    <property name="title">
     <string>控制</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlot"/>
   <addaction name="menuCtrl"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionClear"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/icons8-add-new-64.png</normaloff>:/icon/icon/icons8-add-new-64.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/icons8-file-64.png</normaloff>:/icon/icon/icons8-file-64.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/icons8-save-64.png</normaloff>:/icon/icon/icons8-save-64.png</iconset>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/icons8-export-64.png</normaloff>:/icon/icon/icons8-export-64.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/icons8-play-64.png</normaloff>:/icon/icon/icons8-play-64.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/icons8-pause-64.png</normaloff>:/icon/icon/icons8-pause-64.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/icons8-stop-64.png</normaloff>:/icon/icon/icons8-stop-64.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/icons8-repeat-one-64.png</normaloff>:/icon/icon/icons8-repeat-one-64.png</iconset>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/icons8-save-as-64.png</normaloff>:/icon/icon/icons8-save-as-64.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
